docs/operator-manual/webhook.md

```diff
@@ -56,13 +58,19 @@ data:
 
 stringData:
   # github webhook secret
-  github.webhook.secret: shhhh! it's a github secret
+  webhook.github.secret: shhhh! it's a github secret
```

Comment:
was this incorrect?

Reply:
Yes, there was a typo in the doc regarding the webhook secret options.